src/main/webapp/dashboard-manager.js

```diff
@@ -307,3 +293,23 @@ async function handleLogin() {
     }
   });
 }
+
+function authorizeCallback(authResult) {
+  if (authResult['code']) {
+    $('#signinButton').attr('style', 'display: none');
+    var xhr = new XMLHttpRequest();
```

Comment:
Have you tried to use `fetch`? How was that?

Reply:
now implemented